build.cmd

```diff
@@ -1,10 +1,5 @@
-copy .\src\Stripe\bin\Release\Stripe.net.dll .\build\net45\Stripe.net.dll /Y
-copy .\src\Stripe\bin\Release\Stripe.net.xml .\build\net45\Stripe.net.xml /Y
-copy .\src\Stripe.Portable\bin\Release\Stripe.net.dll .\build\portable\Stripe.net.dll /Y
-copy .\src\Stripe.Portable\bin\Release\Stripe.net.xml .\build\portable\Stripe.net.xml /Y
+dotnet pack -c Release src\Stripe.net
 
-cd build
-del *.nupkg
-nuget pack Stripe.net.nuspec
+copy .\src\Stripe.net\bin\Release\Stripe.net.*.nupkg .\build\
 
```

Comment:
I like the rename to Stripe.net here - however, the default namespace is Stripe, and that should be preserved. If it is, ignore this. Once I merge this in, I'm looking forward to getting this setup on AppVeyor!